main/zoe/guide/contracts/atomic-swap.md

```diff
@@ -57,41 +53,55 @@ const {
 } = await zoe.escrow(aliceOfferRules, alicePayments);
```

Comment:
Let's update this to `const { seat, payout } = await zoe.redeem(newInvite, aliceOfferRules, alicePayments);`

Reply:
Ah, ok, I see you have this part below. In that case, we can delete this call to zoe.escrow entirely.